eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Usage: create_fasta_from_assembly_report.sh </path/to/assembly_report.txt>
+
+if [ "$#" -lt 3 ]
+then
+    echo "Please provide the assembly accession, the path to the assembly report and the output folder."
+    exit 1
+fi
+
+assembly_accession=$1
+assembly_report=$2
+output_folder=$3
+
+for i in `grep -v -e "^#" ${assembly_report} | cut -f5`;
+do
+    echo ${i}
+
+    # Download each GenBank accession in the assembly report from ENA into a separate file
+    # Delete the accession prefix from the header line
+    wget -q -O - "https://www.ebi.ac.uk/ena/data/view/${i}&display=fasta" | sed 's/ENA|.*|//g' > ${i}
+
+    # If a file has more than one line, then it is concatenated into the full assembly FASTA file
+    # (empty sequences can't be indexed)
+    lines=`wc -l ${i} | cut -f1 -d' '`
```

Comment:
no need to read the whole file to check if there are more than two lines, and no need for `cut` if you already ask only for the lines (`-l`).
```suggestion
    lines=`head -n 2 ${i} | wc -l`
```